README.md

```diff
@@ -1 +1,47 @@
-# Newsweek_bootstrap
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+![](https://img.shields.io/badge/Responsive-Design-orange)
+![](https://img.shields.io/badge/HTML-CSS-blue)
+
+# The NextWeek Clone - A Microverse Project
+
+A clone of The NextWeek landing page made for the Microverse FullStack program.
+This is a responsive site and will scale as needed to different devices.
+
+![desktop](screenshots/desktop.PNG)
+
+![tablet](screenshots/tablet.PNG)
+
+![mobile](screenshots/mobile.PNG)
+
+## Built With
+
+- ![](https://img.shields.io/badge/CSS-blue)
+- ![](https://img.shields.io/badge/HTML-red)
+
+## Live Demo
+
+[Live Demo Link](imran5644.github.io/Newsweek_bootstrap/)
+
+##  Getting Started
+
+To run the code simply download the repository by either cloning it or 
+downloading it as Zip file. Then use the old trustable browser to open the HTML file.
```

Comment:
Try to elaborate more steps that a user needs to do after downloading or cloning this project.